pkg/generate/generate.go

```diff
@@ -278,6 +278,7 @@ func applyRule(log logr.Logger, client *dclient.Client, rule kyverno.Rule, resou
 	} else {
 		label["app.kubernetes.io/synchronize"] = "disable"
```

Comment:
We should not be using `app.kubernetes.io` namespace as that is used by standard labels. I think we have used `policy.kyverno.io` in other places.